tools/run_tests/build_artifact_python.sh

```diff
@@ -43,7 +43,11 @@ GRPC_PYTHON_USE_CUSTOM_BDIST=0  \
 GRPC_PYTHON_BUILD_WITH_CYTHON=1 \
 ${SETARCH_CMD} python setup.py  \
     bdist_wheel                 \
-    sdist                       \
+    sdist
```

Comment:
This looks weird and invites the reader to wonder why these commands aren't all together in one statement. How about an apologetic comment linking to a bug report/feature request?